src/backend/dx12/src/device.rs

```diff
@@ -3616,7 +3616,11 @@ impl d::Device<B> for Device {
 
     unsafe fn set_command_buffer_name(&self, command_buffer: &mut cmd::CommandBuffer, name: &str) {
         let cwstr = wide_cstr(name);
-        command_buffer.raw.SetName(cwstr.as_ptr());
+        if !command_buffer.raw.is_null() {
+            command_buffer.raw.SetName(cwstr.as_ptr());
```

Comment:
if we set the name, we still want to remember it in `raw_name` as well, so that we can set it again upon the buffer being re-recorded